examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,239 @@
+    csPlane="XZ", position=[70, 0, -11], dimension_list=[11, 110], name="Coil_terminal"
+)
+
+core = m3d.modeler.create_rectangle(
+    csPlane="XZ", position=[45, 0, -18], dimension_list=[7, 160], name="Core"
+)
+core.sweep_around_axis(cs_axis=AXIS.Z)
+
+region = m3d.modeler.create_region(pad_percent=[20, 20, 500, 20, 20, 100])
+
+###############################################################################
+# Assign materials
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Coil --> copper, core --> ferrite, region --> vacuum
+
+m3d.assign_material(region.name, "vacuum")
+m3d.assign_material(coil.name, "copper")
+m3d.assign_material(core.name, "ferrite")
+
+###############################################################################
+# Assign excitation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Coil consists of 10 turns, coil current 100A.
+
+m3d.assign_coil(["Coil_terminal"], conductor_number=10, name="Coil_terminal")
+
+m3d.assign_winding(
+    is_solid=False,
+    current_value=100,
+    name="Winding1",
+)
+
+m3d.add_winding_coils(windingname="Winding1", coil_names=["Coil_terminal"])
+
+###############################################################################
+# Assign mesh operations
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Not necessary in eddy current solver, but when assigned, they speed up the simulation.
+
+m3d.mesh.assign_length_mesh(
+    ["Core"], maxlength=15, meshop_name="Inside_Core", maxel=None
+)
+m3d.mesh.assign_length_mesh(
+    ["Coil"], maxlength=30, meshop_name="Inside_Coil", maxel=None
+)
+
+###############################################################################
+# Set conductivity temperature coefficient
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Make conductivity function of temperature, resistivity temperature coefficient is 0.393%
+
+Cu = m3d.materials["copper"]
+cu_resistivity_temp_coefficient = 0.00393
+Cu.conductivity.add_thermal_modifier_free_form("1.0/(1.0+{}*(Temp-20))".format(cu_resistivity_temp_coefficient))
+
+###############################################################################
+# Set object temperature and enable feedback
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the temperature of the objects to default temperature (22deg C),
+# and enable temperature feedback for 2-way coupling
+
+m3d.modeler.set_objects_temperature(["Coil"])
+
+###############################################################################
+# Assign matrix
+# ~~~~~~~~~~~~~~~
+# RL calculation
+
+m3d.assign_matrix(["Winding1"], matrix_name="Matrix1")
+
+###############################################################################
+# Create and analyze simulation setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Simulation frequency 150kHz
+
+setup = m3d.create_setup(setupname="Setup1")
+setup.props["Frequency"] = "150kHz"
+m3d.analyze_setup("Setup1")
+
+report = m3d.post.create_report(expressions="Matrix1.R(Winding1,Winding1)")
+solution = report.get_solution_data()
+resistance = solution.data_magnitude()[0]
```

Comment:
just as a suggestion.. :) if you'd like (especially in case of multiple reports) you can specify the expression you'd like to get. For example in your case it would be: 
`resistance = solution.data_magnitude("Matrix1.R(Winding1,Winding1)")[0]`